cmd/cloudshell_open/main.go

```diff
@@ -41,7 +43,7 @@ const (
 	reauthCredentialsWaitTimeout     = time.Minute * 2
 	reauthCredentialsPollingInterval = time.Second
 
-	projectCreateURL = "https://console.cloud.google.com/cloud-resource-manager"
+	instrumentlessEvent = "crbutton"
```

Comment:
if these aren’t pre-registered, can we do cloud-run-button?

Reply:
This one is already registered as `crbutton`.